tests/script/test_add_preload_script.py

```diff
@@ -95,77 +112,42 @@ async def test_preloadScript_add_multipleScripts(websocket, context_id, html):
         websocket, {
             "method": "script.addPreloadScript",
             "params": {
-                "functionDeclaration": "() => { window.foo1=1; }",
+                "functionDeclaration": "() => { "
+                                       "window.SOME_VAR=["
+                                       "...(window.SOME_VAR ?? []), "
+                                       "'PRELOAD_SCRIPT_1']; }",
             }
         })
     await execute_command(
         websocket, {
             "method": "script.addPreloadScript",
             "params": {
-                "functionDeclaration": "() => { window.foo2=2; }",
-            }
-        })
-
-    await goto_url(websocket, context_id, html())
-
-    result = await execute_command(
-        websocket, {
-            "method": "script.evaluate",
-            "params": {
-                "expression": "window.foo1 + window.foo2",
-                "target": {
-                    "context": context_id
-                },
-                "awaitPromise": True,
-                "resultOwnership": "root"
-            }
-        })
-    assert result["result"] == {"type": "number", "value": 3}
-
-
-@pytest.mark.asyncio
-async def test_preloadScript_add_sameScript_multipleTimes(
```

Comment:
Any reason to remove this? `test_preloadScript_add_sameScript_multipleTimes`

The idea was to ensure the implementation is not "taking any shortcut" or caching the scripts in any way, and that they are effectively added twice, and with different IDs.

I think it would be a good idea not to remove this test.
You could refactor it to the way it is here, then we'd have an array with two elements.

Reply:
Ok, added it back